tests/ci_build/Dockerfile.emscripten

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu
+MAINTAINER Aran Khanna <arankhan@amazon.com>
```

Comment:
MAINTAINER is deprecated https://docs.docker.com/engine/reference/builder/#maintainer-deprecated